src/blocks/media-block/ImageBlock.new.test.tsx

```
@@ -0,0 +1,463 @@
+  setZoomState: (value: boolean) => {
+    zoomState.setZoomed(value);
+  },
+  addZoomListener: (cb: () => void) => {
+    zoomState.callbacks.add(cb);
+    return () => zoomState.callbacks.delete(cb);
+  },
+  reset: () => {
+    zoomState._isZoomed = false;
+    zoomState.callbacks.clear();
+  }
+} as const;
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)